app/indexers/etd_indexer.rb

```diff
@@ -44,8 +44,9 @@ def dissertation
 
     # Derive department by stripping "UC, SB" from the degree grantor field
     def department(solr_doc)
-      Array(solr_doc[Solrizer.solr_name("degree_grantor", :symbol)])
```

Comment:
I didn't edit https://github.com/ucsblibrary/alexandria/blob/d92bb888c4bb89676bf16ef64bbaf45b1dd1221f/app/models/etd.rb#L19 but for some reason the Solr field changed from `ssim` to `tesim` and `sim`.

Reply:
Oh, I think it _will_ need to be changed though, since it will need a `_label` field in Solr.

Reply:
Assuming @chrissyrissmeyer wants `degree_grantor` to be a local authority.